src/WebJobs.Script/ManagedDependencies/ManagedDependencyProvider.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.PowerShellWorker.ManagedDependency;
+using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.ManagedDependencies
+{
+    internal class ManagedDependencyProvider : IManagedDependencyProvider
+    {
+        private const string ManagedDependenciesPropertiesName = "managedDependencies";
```

Comment:
Nit: ManagedDependenciesFolder ?

Reply:
I'm reusing this Property as the folder name and the "managedDependencies" property name in host.json are the same.